sagemcom_api/client.py

```diff
@@ -105,7 +105,7 @@ def __init__(
             else ClientSession(
                 headers={"User-Agent": f"{DEFAULT_USER_AGENT}"},
                 timeout=ClientTimeout(DEFAULT_TIMEOUT),
-                connector=TCPConnector(verify_ssl=verify_ssl if verify_ssl else True),
+                connector=TCPConnector(verify_ssl if verify_ssl is not None else True),
```

Comment:
Can you explain why this change is required? Does verify_ssl = False not work?

Reply:
the logic here says something like, `variable=valueA if condition, else valueB`. If `verify_ssl=False`, then it reads, `verify_ssl=False if **False**, else True.` Since False always evaluates to False, we are never able to set the variable to False.

My logic fixes the evaluation, see below example in python interpreter:
`
>>> verify_ssl=False
>>> verify_ssl if verify_ssl else True
True
>>> verify_ssl=True
>>> verify_ssl if verify_ssl else True
True


>>> verify_ssl if verify_ssl is not None else True
True
>>> verify_ssl=False
>>> verify_ssl if verify_ssl is not None else True
False
`

Reply:
Good catch! Makes sense.